lib/std/static_string_map.zig

```diff
@@ -329,20 +261,6 @@ fn testMap(map: anytype) !void {
     try testing.expect(null == map.get("averylongstringthathasnomatches"));
 }
 
-test "void value type, slice of structs" {
-    const slice = [_]TestKVVoid{
-        .{"these"},
-        .{"have"},
-        .{"nothing"},
-        .{"incommon"},
-        .{"samelen"},
-    };
-    const map = TestMapVoid.initComptime(slice);
-    try testSet(map);
-    // Default comparison is case sensitive
-    try testing.expect(null == map.get("NOTHING"));
-}
-
```

Comment:
does this test no longer pass? I don't see this noted in the writeup

Reply:
It should still pass - I must have accidentally left it out. I'll put it back in when I am back to my keyboard.